big_tests/src/ct_markdown_errors_hook.erl

```diff
@@ -161,3 +168,24 @@ reindent(Bin) ->
     %% Use 2 whitespaces instead of 4 for indention
     %% to make more compact look
     binary:replace(Bin, <<"    ">>, <<"  ">>, [global]).
+
+
+get_group_names(Config) ->
+    %% tc_group_path contains something like:
+    %% [[{name,muc_rsm_all},parallel],
+    %%  [{name,rdbms_muc_all},{repeat_until_all_ok,3},parallel]]
+    %% Where muc_rsm_all is subgroup of rdbms_muc_all.
+    Path = proplists:get_value(tc_group_path, Config, []),
+    TopGroup = proplists:get_value(tc_group_properties, Config, []),
+    Names = [ proplists:get_value(name, Props) || Props <- lists:reverse([TopGroup|Path]) ],
+    %% Just in case drop undefined
+    [Name || Name <- Names, Name =/= undefined].
+
+full_group_name(Config) ->
+    %% Groups path, example [main_group, sub_group1, sub_sub_group...]
+    Groups = get_group_names(Config),
+    join_atoms(Groups).
+
+join_atoms(Atoms) ->
+    Strings = [atom_to_list(A) || A <- Atoms],
+    list_to_atom(string:join(Strings, ":")).
```

Comment:
Just being paranoid, I hope we don't run out of space in the atom table

Reply:
we don't